chrome/content/zotero/fileInterface.js

```diff
@@ -129,7 +131,9 @@ var Zotero_File_Interface = new function() {
 	 */
 	function exportFile() {
 		var exporter = new Zotero_File_Exporter();
-		exporter.name = Zotero.getString("pane.collections.library");
+		exporter.libraryID = ZoteroPane_Local.getSelectedLibraryID();
+		if exporter.libraryID === false throw('No library selected');
```

Comment:
Dammit, the extension pointed to a different repo checkout. I've nuked all but one version.